src/types/Appstream.ts

```diff
@@ -105,3 +106,7 @@ export interface Bundle {
   runtime: string
   sdk: string
 }
+
+export interface Metadata {
+  'Flathub::manifest': string
```

Comment:
this should also be 
```suggestion
  'Flathub::manifest'?: string
```
because it is optional iirc

Reply:
Addressed in a separate commit just now.